src/test/java/run/halo/app/controller/content/api/PostControllerTest.java

```diff
@@ -0,0 +1,84 @@
+package run.halo.app.controller.content.api;
+
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import run.halo.app.model.dto.BaseCommentDTO;
+import run.halo.app.model.entity.PostComment;
+import run.halo.app.model.params.PostCommentParam;
+import run.halo.app.service.OptionService;
+import run.halo.app.service.PostCommentService;
+import run.halo.app.service.PostService;
+
+@SpringBootTest
+@ActiveProfiles("test")
+public class PostControllerTest {
```

Comment:
Hi @jerry-shao , could you remove this `PostControllerTest`? Because you have provided `PostAPITest` already.

Reply:
Removed.